actors/builtin/verifreg/verified_registry_test.go

```diff
@@ -20,32 +20,74 @@ func TestExports(t *testing.T) {
 }
 
 func TestConstruction(t *testing.T) {
-	actor := verifreg.Actor{}
 	receiver := tutil.NewIDAddr(t, 100)
-	builder := mock.NewBuilder(context.Background(), receiver).
-		WithCaller(builtin.SystemActorAddr, builtin.InitActorCodeID)
 
-	rt := builder.Build(t)
+	runtimeSetup := func() *mock.Runtime {
+		builder := mock.NewBuilder(context.Background(), receiver).
+			WithCaller(builtin.SystemActorAddr, builtin.InitActorCodeID)
+		rt := builder.Build(t)
```

Comment:
This is incorrect: the `t` must come from the `t.Run`, not the outer scope.

You could pass it as an arg, but it's just confusing. Do make the builder outside, but then just do `builder.Build(t)` inside `t.Run`.